tools/testsys/src/crds.rs

```diff
@@ -315,6 +324,84 @@ impl<'a> CrdInput<'a> {
 
         Ok(Some(rendered_config))
     }
+
+    /// Find the hardware csv file for the given hardware csv name and test type.
+    fn hardware_csv_file_path(&self, hardware_csv: &str) -> Option<PathBuf> {
```

Comment:
This seeking for a required file seems a bit complicated. Will it be clear to the user how this works and what went wrong when the desired file isn't found?

Reply:
There are a few supported ways to organize the files within the `tests` directory. I will include that in TESTING.md.